### PR TITLE
Hopefully fixes THAT ONE BUG

### LINKED ISSUE #18600: Ratvar'd cyborgs don't always properly get the antag hud
[Note]: # (State what the issue is from a "whats wrong" prospective. Issue reports should clearly allow maintainers to understand whats wrong and how to test/reproduce if that is not obvious. Avoid ambiguity. Start your issue report below both of these lines (or remove them))

[Admins]: # (If you are reporting a bug that occured AFTER you used varedit/admin buttons to alter an object out of normal operating conditions, please verify that you can re-create the bug without the varedit usage/admin buttons before reporting the issue.)
Two cyborgs got converted and both failed to get the antag hud, one of those also failed to get antag laws and emagged items.

### PR DESCRIPTION
**Pull request by ChangelingRain**
Fixes #18600 hopefully

Downside; servants debrained and placed into a borg shell won't get autoconverted.

### CODE REVIEW

File: code/game/gamemodes/game_mode.dm [hunk 1]
```diff
@@ -536,3 +536,4 @@
 	ticker.mode.remove_revolutionary(newborgie, 0)
 	ticker.mode.remove_gangster(newborgie, 0, remove_bosses=1)
 	ticker.mode.remove_hog_follower(newborgie, 0)
+	remove_servant_of_ratvar(newborgie.current, TRUE)
```
Comment (KorPhaeron) [inline code/game/gamemodes/game_mode.dm]: ticker.mode.remove?

Comment (ChangelingRain) [inline code/game/gamemodes/game_mode.dm]: nope, it's not a ticker.mode proc

**Comment by Xhuis** [comment]
This might break soul vessels.

**Comment by ChangelingRain** [comment]
It shouldn't; it'll remove their antag, but then return it if the MMI was clockwork, and the removal is silent so they should only get the 'yer a servant larry' messages.

File: code/game/gamemodes/game_mode.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/game/gamemodes/game_mode.dm]: thats odd

**Pull request merged**